Gemfile

```diff
@@ -53,7 +53,7 @@ end
 group :development do
   gem "bullet"
   gem "listen"
-  gem "rubocop-rails_config"
+  gem "rubocop-rails_config", '0.3.0.rc1'
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.